django/test/runner.py

```diff
@@ -525,6 +528,11 @@ def add_arguments(cls, parser):
             '--no-faulthandler', action='store_false', dest='enable_faulthandler',
             help='Disables the Python faulthandler module during tests.',
         )
+        parser.add_argument(
+            '--timing', action='store_true',
+            help='Output timings such as database creation, cloning, and teardown;'
+            ' and total test run time.',
```

Comment:
I'm not sure why we have here a semicolon :thinking:, please use also hanging indentation:
 ```suggestion
            help=(
                'Output timings such as database creation, cloning, teardown, '
                ' total test run time.',
            ),
```

Reply:
That came from my suggestion. Originally it was `DB creation, DB cloning, DB teardown and total test run time` and, aside from `DB` → `database`, it seemed unnecessary to repeat `database` multiple times.

Semicolons can be used in complex lists where there are commas used within the list items themselves. I perceived there to be two list items here:

- database creation, cloning, and teardown
- total test run time

Your suggested wording now no longer makes it clear that "cloning" and "teardown" are database-related.